FlowCrypt/Controllers/Inbox/InboxViewController.swift

```diff
@@ -452,10 +452,22 @@ extension InboxViewController: MsgListViewController {
                 tableNode.reloadData()
             } else {
                 state = .fetched(.byNumber(total: newTotalNumber))
-                tableNode.deleteRows(at: [IndexPath(row: index, section: 0)], with: .left)
+                do {
+                    try ObjException.catch {
+                        self.tableNode.deleteRows(at: [IndexPath(row: index, section: 0)], with: .left)
+                    }
+                } catch {
+                    logger.logError("Failed to remove message at \(index) in fetched state")
```

Comment:
Let's also add catched error message here, so it'll be `"Failed to remove message at \(index) in fetched state: \(error)"`